adapters/beachfront/beachfront.go

```diff
@@ -499,6 +527,13 @@ func extractVideoCrid(nurl string) string {
 	return strings.TrimSuffix(chunky[2], ":")
 }
 
+// Thank you, brightroll.
```

Comment:
Can I ask what's the purpose of this comment? If it's just a leftover of some sort, can we please remove it? Thanks!